tfhe-zk-pok/src/curve_api.rs

```diff
@@ -164,6 +180,9 @@ impl FieldOps for bls12_381::Zp {
     fn hash(values: &mut [Self], data: &[&[u8]]) {
         Self::hash(values, data)
     }
+    fn hash_128bit(values: &mut [Self], data: &[&[u8]]) {
```

Comment:
why the adition ?

Reply:
because according to Benoit we could use a 128 bit exponent for the `t_i`, so i added a new hash function that generates those

Reply:
is it used currently ?

Reply:
yeah, it's used for the pke proof